lib/session.js

```diff
@@ -85,6 +96,12 @@ module.exports = class Session {
           this[cookieSignerKey]
         )
 
+        if (Array.isArray(keys)) {
+          for (const key of keys) {
+            session.set(key, this[key])
+          }
+        }
```

Comment:
I don't really like this code duplication. I think we should move the whole `new Session` block outside of the if-else.